### ISSUE TITLE
Form element layout parameter

### ISSUE THREAD

**Issue by advation** [Staple-Code/Staple-Code#23]
In FormRefactoring establish a parameter that defines if a forms elements should be stacked vertically or aligned horizontally.

**Comment by hansheeling** [comment]
Great Idea, I am curious if though we want to leave this as a custom field view option maybe?

**Comment by ironpilot** [comment]
I think this should be an option that is linked with the view adapter piece just to make sure that it is fully customizable for other styles as well.

**Comment by advation** [comment]
I like both ideas if it's something that we can work out. It would be nice to have this functionality available for both an entire form structure as well as individual elements. For the entire form I'd like to be able to set an option like **setOrientation('vertical')** when I'm setting up a form. 

```
class testForm extends Form
{
    public function _start()
    {
        $this->setName('test')
            ->setAction($this->link(array('test','index')))
            ->setOrientation('vertical');
    }
}
```

I'm not sure how this would look for individual element layouts. Any ideas?

**Comment by ironpilot** [comment]
Interesting idea. I do like the flow idea that you have. What do you think about something like this:

```
class testForm extends Form
{
    public function _start()
    {
        $this->setName('test')
            ->setAction($this->link(array('test','index')))
            ->setViewOptions('orientation-vertical');
    }
}
```

This way would would be able to add even more options that could even be specific to each view adapter. I would just like to keep as much isolated to the implementation of the specific view adapter as possible.

**Comment by advation** [comment]
That looks great to me! Would this be applicable to a specific field element as well?

**Comment by ironpilot** [comment]
Yeah I think that's a good idea to propagate that down to each element as well and have that customization option as well.
